pkg/sfu/downtrack.go

```diff
@@ -239,7 +236,7 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 // because a track has been stopped.
 func (d *DownTrack) Unbind(_ webrtc.TrackLocalContext) error {
 	d.bound.set(false)
-	d.receiver.DeleteDownTrack(d.peerID)
+	d.receiver.DeleteDownTrack(d.ID())
```

Comment:
TrackID in lk-server is global unique, there is no need of peerID

Reply:
It's globally unique per *track*, but not unique across downtracks.. For example, an UpTrack may come in as `TR_123`, all of the downtracks will inherit that track id. We'll still need to use the peerID to identify which peer the downtrack is for.

Reply:
misunderstand it. revert change of peerid